test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Culture.Tests.ps1

```diff
@@ -1,25 +1,64 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
-Describe "Get-Culture DRT Unit Tests" -Tags "CI" {
-    It "Should works proper with get-culture" {
-        $results = get-Culture
-        $results -is "System.Globalization.CultureInfo" | Should -BeTrue
-        $results[0].Name | Should -Be $PSCulture
-    }
-}
 
 Describe "Get-Culture" -Tags "CI" {
 
     It "Should return a type of CultureInfo for Get-Culture cmdlet" {
 
-	Get-Culture | Should -BeOfType CultureInfo
+        $culture = Get-Culture
+        $culture | Should -BeOfType [CultureInfo]
+        ($culture).LCID | Should -Be $host.CurrentCulture.LCID
+
+        Get-Culture -NoUserOverrides | Should -BeOfType [CultureInfo]
+    }
+
+    It "Should have `$PSCulture variable be equivalent to (Get-Culture).Name" {
+
+        (Get-Culture).Name | Should -Be $PsCulture
```

Comment:
Based on the test description, it would seem that this should be:

> $PSCulture | Should -BeExactly (Get-Culture).Name

Reply:
I think the order is right because the file is for Get-Culture tests and we should correct the description.
But I'd expect that we check $Host.CurrentCulture instead of `$PsCulture`.
On other side, we haven't tests for special variables at all.

What do you think?

Reply:
I'm fine changing the test description to match the test and adding separate test for $PSCulture

Reply:
Fixed.